src/confirm-page.html

```diff
@@ -23,8 +23,21 @@ <h1 class="title-text" data-i18n-message-id="openThisSiteConfirmation"></h1>
       </label>
       <br />
       <div class="button-container">
-        <button id="deny" class="button" data-message-id="openInContainer" data-message-arg="current-container-name"></button>
-        <button id="confirm" class="button primary" autofocus data-message-id="openInContainer" data-message-arg="container-name"></button>
+        <button id="deny"
+                class="button"
+                data-message-id="openInContainer"
+                data-message-arg="current-container-name">
+        </button>
+        <button id="deny-no-container"
+                class="button"
+                data-message-id="openInNoContainer">
```

Comment:
I checked out this branch and did a `git submodule update` but this message ID still isn't in the `src/_locales/en/messages.json` file. Does this branch need another submodule udpate?

Reply:
Something's wrong with `git submodule update`. It always checkout to an old commit (`Submodule path 'src/_locales': checked out '26e17db70ce276ab5ccb5ae582e5c142f12ba655'`)

Reply:
Looks like this is fixed now!